cores/esp8266/core_esp8266_phy.c

```diff
@@ -31,7 +31,7 @@
 static const uint8_t ICACHE_FLASH_ATTR phy_init_data[128] =
 {
     [0] = 5,  // Reserved, do not change
-    [1] = 0,  // Reserved, do not change
+    [1] = 8,  // Reserved, do not change
```

Comment:
do we know what this byte does?

Reply:
According to https://github.com/espressif/ESP8266_NONOS_SDK/commit/d62e5a70d894028066bfe47dfd6e2a8a39dd5483, second byte is some version number.